jsonnet/node-exporter.jsonnet

```diff
@@ -72,13 +73,31 @@ local tlsVolumeName = 'node-exporter-tls';
       },
 
     // This configures the kube-rbac-proxies to use the serving cert
-    // configured on the `Service` above.
+    // configured on the `Service` above and adds the default init text
+    // collectors to the process.
 
     daemonset+:
       {
         spec+: {
           template+: {
             spec+: {
+              initContainers+: [
+                {
+                  name: 'init-textfile',
+                  command: ['/bin/sh', '-c', '[[ ! -d /node_exporter/collectors/init ]] || find /node_exporter/collectors/init -perm /111 -type f -exec {} \\;'],
+                  env: [{name: "TMPDIR", value: "/tmp"}],
+                  image: $._config.imageRepos.nodeExporter + ':' + $._config.versions.nodeExporter,
+                  resources: {},
+                  securityContext: {
+                    runAsUser: 0,
+                  },
+                  terminationMessagePolicy: 'FallbackToLogsOnError',
+                  volumeMounts+: [
+                    containerVolumeMount.new(textfileVolumeName, '/etc/node_exporter/textfile'),
+                  ],
+                  workingDir: '/etc/node_exporter/textfile',
```

Comment:
Let's adhere to `man hier(7)` and put this into `/var` as those are not config files. This is a common practice when using textfile collector.

Reply:
`/var/node_exporter/textfile`?

Reply:
sgtm